examples/flutter_gallery/lib/demo/animation/home.dart

```diff
@@ -202,15 +202,14 @@ class _AllSectionsLayout extends MultiChildLayoutDelegate {
     // When tCollapsed > 0, the indicators move closer together
     //final double rowIndicatorWidth = 48.0 + (1.0 - tCollapsed) * (rowTitleWidth - 48.0);
     const double paddedSectionIndicatorWidth = kSectionIndicatorWidth + 8.0;
-    final double rowIndicatorWidth = paddedSectionIndicatorWidth +
-      (1.0 - tCollapsed) * (rowTitleWidth - paddedSectionIndicatorWidth);
+    final double rowIndicatorWidth =
+        paddedSectionIndicatorWidth + (1.0 - tCollapsed) * (rowTitleWidth - paddedSectionIndicatorWidth);
```

Comment:
I think the original wrapping is mode readable.

Generally I would avoid breaking after an assignment operator above more or less everything else.

Reply:
Here again a trailing comment can help the formatter:

```dart
    final double rowIndicatorWidth = paddedSectionIndicatorWidth + //
        (1.0 - tCollapsed) * (rowTitleWidth - paddedSectionIndicatorWidth);
```

I do thing _splitting_ expression is really hard. You will always find case where the current formatting is not optimal. The hard thing is to find the best general formatting.